libpod/container_internal_linux.go

```diff
@@ -440,7 +440,7 @@ func (c *Container) getOCICgroupPath() (string, error) {
 }
 
 func openDirectory(path string) (fd int, err error) {
-	return unix.Open(path, unix.O_RDONLY|unix.O_PATH, 0)
+	return unix.Open(path, unix.O_RDONLY|unix.O_PATH|unix.O_CLOEXEC, 0)
```

Comment:
IT would be good to split this into its own commit since it is not directly related the the pause image change

Reply:
Done in https://github.com/containers/podman/pull/25879. I will keep it also here for limited time so the tests do not fail.